transport/rif_comms/node.py

```diff
@@ -204,7 +202,6 @@ def stop(self):
         if self.greenlet:
             self.greenlet.kill()
             self.greenlet.get()
-        self.greenlet = None
```

Comment:
i have removed this because on the line 210 we wait for the greenlet and it was none

i dont think the assignment make any sense since the node is shutting down anyway (and the memory is released)

Reply:
i'm not sure where we got this from. i'm positive it was copied over from somewhere else, but i can't seem to find it anymore :man_shrugging:

Reply:
this was on the matrix client

Reply:
> this was on the matrix client

you're right, but it doesn't really add up then.

the matrix node, when stopped, calls a function for its matrix client, which kills its thread.

shouldn't the rif comms node do something analogous?

Reply:
@marcosmartinez7 can you confirm whether this is happening or not?

Reply:
the client is not running on a thread so we don't actually need this, resolving